src/views/addManager/addManager.js

```diff
@@ -73,12 +73,23 @@ export const AddManager = () => {
       ...data,
       password: 'changeAt1stLogin!',
       confirmPassword: 'changeAt1stLogin!',
-      // role: RoleEnum.PROPERTY_MANAGER,
-      // propertyIDs: propertySelection.map(p => p.key),
+      role: "PROPERTY_MANAGER",
```

Comment:
We should be able to use the global enum instead of hard-coding the role here. Something like: `RoleEnum.PROPERTY_MANAGER`